packages/core/contracts/financial-templates/common/FeePayer.sol

```diff
@@ -138,6 +134,7 @@ abstract contract FeePayer is AdministrateeInterface, Testable, Lockable {
         _adjustCumulativeFeeMultiplier(totalPaid, collateralPool);
 
         if (regularFee.isGreaterThan(0)) {
+            StoreInterface store = _getStore();
```

Comment:
this is moved to here has the store is not needed sooner in this method. save a small bit of gas on calls.